### PR TITLE
NP-1500: update nva-commons

### PR DESCRIPTION
**Pull request by axthosarouris**
Necessary for using the newest serialization settings

### CODE REVIEW

File: find-doi-requests/src/main/java/no/unit/nva/doi/requests/FindDoiRequestsHandler.java [hunk 1]
```diff
@@ -84,6 +82,11 @@ protected DoiRequestsResponse processInput(Void input, RequestInfo requestInfo,
         return DoiRequestsResponse.of(doiRequests);
     }
 
+    @Override
```
Comment (axthosarouris) [inline find-doi-requests/src/main/java/no/unit/nva/doi/requests/FindDoiRequestsHandler.java:85]: moved from line 112, due to auto-formatting

**Review by brinxmat — APPROVED** [review]

**Comment by norrs** [comment]
I've already done this in https://github.com/BIBSYSDEV/nva-doi-requests-api/pull/28 FYI, rather get this PR merged but @brinxmat left some feedback on friday afternoon that I fixed now this morning.

**Comment by norrs** [comment]
FYI @axthosarouris

**Review by norrs — CHANGES_REQUESTED** [review]
We dont need this PR, but if you want to run a auto-reformat on the whole class, go a head and run it and push and update and rename title on PR.

**Pull request closed without merging**